TMS.Net07.Homework.DaysOfWeek/middle.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TMS.Net07.Homework.DaysOfWeek
+{
+    class middle
```

Comment:
```suggestion
    class Middle
```

In C# we usually name classes and methods from upper-case letter

Reply:
done